ui/src/pages/AllStories/index.tsx

```diff
@@ -208,6 +227,56 @@ export const AllStories: React.FC = () => {
     dispatch({ type: "SET_TAB_VALUE", newValue });
   };
 
+  const requestSearchHelper = (row, searchedVal: string) => {
+    let doesExist = false;
+    Object.keys(row).forEach((prop) => {
+      //Exclude search for StoryView members not displayed on table cells
+      const excludedParameters =
```

Comment:
@nimwijetunga We don't want it to show stories that may include the search keywords inside of its content right? I.e. they search "Toronto" and a story mentions Toronto in their details so then it'll show up in the valid stories

Reply:
Yeah lets exclude any fields that are not directly visible on the table